internal/listener/listener.go

```diff
@@ -133,7 +134,16 @@ func (l *Listener) ProcessEvent(w http.ResponseWriter, req *http.Request) {
 
 	l.logger.Infow("Processing event", zap.String("event", ev.String()))
 	err = incident.ProcessEvent(context.Background(), l.db, l.logs, l.runtimeConfig, &ev)
-	if err != nil {
+	switch {
+	case errors.Is(err, event.ErrSuperfluousStateChange):
+		abort(http.StatusAlreadyReported, &ev, "Stopped processing event with superfluous state change: %v", err)
+		return
+
+	case errors.Is(err, event.ErrSuperfluousMuteUnmuteEvent):
+		abort(http.StatusAlreadyReported, &ev, "Stopped processing event with superfluous (un)mute object: %v", err)
```

Comment:
The messages given here are somewhat redundant with what's already part of `err.Error()`, so this results in responses like this:

```
Stopped processing event with superfluous state change: ignoring superfluous state change: ok state event from source 1
```

Reply:
I have unified both cases and picked another <small>arbitrary</small> HTTP status code. Now the returned message looks like:
> ignoring superfluous state change: ok state event from source 2